tests/next_tests/integration_tests/feature_tests/ffront_tests/ffront_test_utils.py

```diff
@@ -78,7 +86,7 @@ def fieldview_backend(request):
             skip_mark(msg.format(marker=marker, backend=backend_id))
 
     backup_backend = decorator.DEFAULT_BACKEND
-    decorator.DEFAULT_BACKEND = no_backend
+    decorator.DEFAULT_BACKEND = no_backend()
```

Comment:
```suggestion
    decorator.DEFAULT_BACKEND = no_backend
```
Superfluous after the suggested change above.